cmd/gateway-azure_test.go

```diff
@@ -82,6 +82,14 @@ func TestS3MetaToAzureProperties(t *testing.T) {
 			t.Fatalf("Test failed with unexpected error %s, expected UnsupportedMetadata", err)
 		}
 	}
+
+	headers = map[string]string{
+		"content-md5": "Dce7bmCX61zvxzP5QmfelQ==",
+	}
+	_, props, _ := s3MetaToAzureProperties(headers)
```

Comment:
You would need to get and check error also

Reply:
Will do tonight